aipcli/command.go

```diff
@@ -58,21 +59,25 @@ func NewMethodCommand(
 				return err
 			}
 		}
-		conn, err := dial(cmd.Context())
-		if err != nil {
-			return err
-		}
-		LogRequest(cmd.Context(), in)
-		if err := conn.Invoke(cmd.Context(), methodURI(method), in, out); err != nil {
-			LogError(cmd.Context(), err)
-			os.Exit(1)
-		}
-		LogResponse(cmd.Context(), out)
-		return nil
+		return invoke(cmd.Context(), methodURI(method), in, out)
 	}
 	return cmd
 }
 
+func invoke(ctx context.Context, uri string, request, response proto.Message) error {
+	conn, err := dial(ctx)
+	if err != nil {
+		return err
+	}
+	logRequest(ctx, request)
+	if err := conn.Invoke(ctx, uri, request, response); err != nil {
+		logError(ctx, err)
+		os.Exit(1)
```

Comment:
what is the reason to not return the error here?

Reply:
I think it was to avoid getting additional error exit boilerplate - I think we can remove this now

Reply:
avoid getting noisy error console output that is

Reply:
great